tests/InputFiles/GeneralizedHarmonic/GaugeWave.yaml

```diff
@@ -0,0 +1,96 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: EvolveGhGaugeWave
+# Check: parse;execute
+# Timeout: 8
+# ExpectedOutput:
+#   GhGaugeWaveVolume0.h5
+#   GhGaugeWaveReductions.h5
+
+Evolution:
+  InitialTime: 0.0
+  InitialTimeStep: 0.0002
+  TimeStepper:
+    AdamsBashforthN:
+      Order: 3
```

Comment:
Why order 3 instead of order 1?

Reply:
I had the impression that order 3 was more typically recommended for evolutions, so it seemed better to put it in the example -- AB1 is just forward Euler, I think.